seisflows/workflow/inversion.py

```diff
@@ -270,12 +287,21 @@ def evaluate_initial_misfit(self):
                      self.evaluate_objective_function],
                     path_model=path_model,
                     save_residuals=os.path.join(self.path.eval_grad,
-                                                "residuals.txt")
+                                                "residuals_{src}_{it}_0.txt")
```

Comment:
I think here we can explicitly name the iteration since we have access to `self.iteration`.
```python
                # Run forward simulation with previous model. Hard set line search
                # step count in residual file names to 0 since it is assumed we are 
                # running the initial misfit evaluation (i??s00)
                self.system.run(
                    [self.run_forward_simulations,
                     self.evaluate_objective_function],
                    path_model=path_model,
                    save_residuals=os.path.join(self.path.eval_grad,
                                                f"residuals_{{src}}_{self.iteration}_0.txt")
````